CMakeLists.txt

```diff
@@ -17,10 +17,21 @@ if(PROJECT_IS_TOP_LEVEL)
 
   option(BUILD_CPP "Build C++ library" ON)
 
+  option(aws-greengrass-sdk-lite_BUILD_TESTING "Build C/C++ testing" ON)
```

Comment:
```suggestion
  option(BUILD_TESTING "Build C/C++ testing" ON)
```
BUILD_TESTING is a standard thing
https://cmake.org/cmake/help/latest/variable/BUILD_TESTING.html

Reply:
Can just use it (we're only using it if we are top level)